configurator/src/main/resources/stateSetsTabView.fxml

```diff
@@ -43,18 +42,16 @@
                       <ColumnConstraints hgrow="SOMETIMES" maxWidth="411.0" minWidth="10.0" prefWidth="396.0" />
                     </columnConstraints>
                     <rowConstraints>
-                      <RowConstraints maxHeight="26.0" minHeight="26.0" prefHeight="26.0" vgrow="SOMETIMES" />
-                      <RowConstraints maxHeight="26.0" minHeight="26.0" prefHeight="26.0" vgrow="SOMETIMES" />
-                      <RowConstraints maxHeight="26.0" minHeight="26.0" prefHeight="26.0" vgrow="SOMETIMES" />
-                        <RowConstraints maxHeight="26.0" minHeight="0.0" prefHeight="39.0" vgrow="SOMETIMES" />
+                      <RowConstraints maxHeight="26.0" minHeight="0.0" prefHeight="0.0" vgrow="SOMETIMES" />
+                      <RowConstraints maxHeight="52.0" minHeight="26.0" prefHeight="31.0" vgrow="SOMETIMES" />
+                      <RowConstraints maxHeight="47.0" minHeight="19.0" prefHeight="39.0" vgrow="SOMETIMES" />
+                        <RowConstraints maxHeight="54.0" minHeight="0.0" prefHeight="34.0" vgrow="SOMETIMES" />
                         <RowConstraints maxHeight="551.0" minHeight="15.0" prefHeight="485.0" valignment="TOP" vgrow="SOMETIMES" />
                         <RowConstraints maxHeight="274.0" minHeight="0.0" prefHeight="117.0" vgrow="SOMETIMES" />
                     </rowConstraints>
                      <children>
-                        <Label text="Name" />
                         <Label text="Vendor Name" GridPane.rowIndex="1" />
                         <Label text="Vendor IANA" GridPane.rowIndex="2" />
-                        <TextField fx:id="stateSetNameTextField" disable="true" editable="false" GridPane.columnIndex="2" />
                         <TextField fx:id="stateSetVendorNameTextfield" onAction="#onStateSetVendorNameAction" GridPane.columnIndex="2" GridPane.rowIndex="1" />
                         <TextField fx:id="stateSetVendorIANA" onAction="#onStateSetVendorIANAAction" GridPane.columnIndex="2" GridPane.rowIndex="2" />
```

Comment:
If you put all of the TextFields sequentially it will fix tabbing. Presently, if you tab after Vendor IANA textfield, it focuses the labels because those are what are right below  in the fxml.